Framework/Algorithms/src/PhaseQuadMuon.cpp

```diff
@@ -229,43 +233,53 @@ PhaseQuadMuon::squash(const API::MatrixWorkspace_sptr &ws,
   // Get the maximum asymmetry
   double maxAsym = 0.;
   for (size_t h = 0; h < nspec; h++) {
-    if (phase->Double(h, asymmetryIndex) > maxAsym) {
+    if (phase->Double(h, asymmetryIndex) > maxAsym &&
+        phase->Double(h, asymmetryIndex) != 999.0) {
```

Comment:
What's special about `999.0`? Could it ever happen that `phase->Double(h, asymmetryIndex)` is legitimately `999.0`?

Reply:
nope. Typically the values have a modulus of about 0.3. After talking to the scientists it was agreed that 999 will clearly indicate something has gone wrong

Reply:
That's cool - just making sure!

Reply:
Could we factor out `999.0` and give it a name like you've done above to be clear what this value is?

Reply:
ah, I missed this one